core/services/job_services.py

```diff
@@ -0,0 +1,119 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains functions used for polling, training and saving jobs."""
```

Comment:
I don't think so. They are just using the `remote_access_service` functions, so as long as they are working these functions should work fine, too.

(I wasn't going to add this layer initially but later on I added it because higher modularity is always good for future maintenance)